packages/calcite-components/src/utils/globalAttributes.ts

```diff
@@ -1,8 +1,8 @@
 import { createObserver } from "./observers";
 
 type AttributeObject = { [k: string]: any };
-type AllowedGlobalAttribute = "lang" | "role";
-const allowedGlobalAttributes = ["lang", "role"];
+type AllowedGlobalAttribute = "lang" | "role" | "aria-expanded";
```

Comment:
Popover also sets `aria-controls`, so does that also need to be set here? Not sure if we need `aria-describedby` to support tooltip. cc @geospatialem

Reply:
FWIW, `aria-controls` need not be parsed on to the native button for the following reasons:
-It has minimal support across screen readers in terms of providing context as per https://a11ysupport.io/tech/aria/aria-controls_attribute. 
- For scenarios where screen reader should jump directly onto the opened element, the popover is currently doing that with or without aria-controls.

Reply:
In VoiceOver, context is provided when user check/uncheck the checkbox before closing the popup.

Reply:
Windows OS + JAWS updates this week mitigated the issue from last week. 💪🏻